apps/frontend/src/network/__tests__/Network.test.tsx

```diff
@@ -130,10 +118,14 @@ describe('the network page', () => {
       const toggle = getByText(/teams/i, { selector: 'nav a *' });
       fireEvent.click(toggle);
       expect(searchBox.value).toEqual('test123');
-      await waitFor(() => {
-        const { searchParams } = new URL(teamsReq!.path, 'http://api');
-        expect(searchParams.get('search')).toBe('test123');
-      });
+      await waitFor(() =>
+        expect(mockGetTeams).toHaveBeenLastCalledWith(
+          expect.not.objectContaining({
+            filters: expect.anything(),
```

Comment:
So I am checking filters isn't set but I would also like to check search query is set. Any idea if i can do both together?

Reply:
tbh, just get the object out `const [[options]] = mockGetTeams.mock.calls.slice(-1)` and make separate assertions on it. That'll be the most readable I suspect